recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.0.2" %}
+
+package:
+    name: pyamg
+    version: {{ version }}
+
+source:
+    fn: pyamg-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyamg/pyamg-{{ version }}.tar.gz
+    md5: bd024253ebae6f7bf2d89fe32fc5d64a
+
+build:
+    number: 0
+    script: python setup.py install
```

Comment:
And the build errors are from the extra ``setup.py`` commands @jakirkham asked me to add, which don't work with numpy's distutils. I've removed them, and now everything seems to be fine.